Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/NativeArrayTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using NUnit.Framework;
+using Unity.Collections;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class NativeArrayTests : TypeTesterBase
+    {
+        public static (int[] array, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'representations' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)